profiles/rasterbot.lua

```diff
@@ -0,0 +1,42 @@
+-- Rasterbot profile
+
+-- Minimalist node_ and way_functions in order to test source_ and segment_functions
+
+function node_function (node, result)
+end
+
+function way_function (way, result)
+  local highway = way:get_value_by_key("highway")
+  local name = way:get_value_by_key("name")
+
+  if name then
+    result.name = name
+  end
+
+  result.forward_speed = 15
+  result.backward_speed = 15
+end
+
+function source_function ()
+  raster_source = sources:load(
+    "../test/fixtures/rastersource.asc",
```

Comment:
but still, it means hardcoding paths in code. each time you move a file you then need to change your profile code. i think moving paths to something like a settings file would be nice. eg:

# raster_source.ini
elevation /data/raster_4w3452.asc
pollution /data/raster_84hhw.asc
layer2 /data/raster_cnn322.asc